docs/web-ui.md

```diff
@@ -49,6 +49,50 @@ sizes and using executors for all partitions in an RDD or DataFrame.
 The Environment tab displays the values for the different environment and configuration variables,
 including JVM, Spark, and system properties.
 
+<p style="text-align: center;">
+  <img src="img/webui-env-tab.png"
+       title="Env tab"
+       alt="Env tab"
+       width="100%" />
+  <!-- Images are downsized intentionally to improve quality on retina displays -->
+</p>
+
+We can see that this environment page has five parts, it is a useful place to check whether your properties have
+been set correctly. The first part 'Runtime Information' simply contains the
+[runtime properties](configuration.html#runtime-environment) like versions of Java and Scala.
+The second part 'Spark Properties' lists the [application properties](configuration.html#runtime-environment) like
```

Comment:
is this the correct link? 
configuration.html#application-properties

Reply:
good catch!